src/parallel-rustc.md

```diff
@@ -1,34 +1,110 @@
 # Parallel Compilation
 
-As of <!-- date-check --> May 2022, The only stage of the compiler
-that is already parallel is codegen. The nightly compiler implements query evaluation,
-but there is still a lot of work to be done. The lack of parallelism at other stages
-also represents an opportunity for improving compiler performance. One can try out the current
-parallel compiler work by enabling it in the `config.toml`.
+As of <!-- date-check --> August 2022, the only stage of the compiler that 
+is already parallel is codegen. Some other parts of the nightly compiler 
```

Comment:
"nightly" here feels odd to me; we don't ship or test this configuration (just for successful compilation, no more).

I think dropping nightly or otherwise making that distinction clear is worthwhile.